RELEASE-NOTES.txt

```diff
@@ -1,5 +1,6 @@
 13.0
 -----
+* Block editor: Added the ability to enable/disable the block editor on a per site basis.
```

Comment:
How about we also add `Block editor: Default to the block editor unless user opted out in v12.9`?

Reply:
Cool! Added in 4ba301608db9cf86e81fab13191903f3684ea0d5.